detectable/src/test/java/com/synopsys/integration/detectable/detectables/go/functional/GoModDetectableTest.java

```diff
@@ -28,11 +28,17 @@ protected void setup() throws IOException {
         addFile(Paths.get("go.mod"));
 
         ExecutableOutput goListOutput = createStandardOutput(
```

Comment:
I have modified this test as well as `GoModMinusWhyDetectableTest` to use the new json output we expect, as well as fixed the tests . This test had what I presumed to be a bad relationship and a lot of missing assertions.

I have constructed two new Detectable tests, `GoModDetectableDepVerifyTest` and `GoModDetectableDepVerifyMinusWhyTest` from an actual example project that I created.

I don't think there is much value in having the original tests around despite me fixing them given that I think we have a better replacement.

Reply:
I fully support deleting tests that don't provide value!

Reply:
Yeah, seems like GoModDetectableTest and GoModMinusWhyDetectableTest are probably redundant